packages/wonder-blocks-grid/components/gutter.js

```diff
@@ -1,16 +1,25 @@
 // @flow
 import * as React from "react";
+import {
+    MediaLayout,
+    Strut,
+    queryMatchesSize,
+} from "@khanacademy/wonder-blocks-layout";
+import type {MediaQuery} from "@khanacademy/wonder-blocks-layout";
 
-import {MediaLayoutWrapper} from "@khanacademy/wonder-blocks-core";
-import type {MediaSize, MediaSpec} from "@khanacademy/wonder-blocks-core";
-
-import {matchesSize} from "../util/utils.js";
-import FixedWidthCell from "./fixed-width-cell.js";
+type Props = {|
+    /**
+     * Which media should this cell be renderer on.  Defaults to all.
+     */
+    mediaQuery: MediaQuery,
+|};
 
 /**
- * Gutter is a form of [FixedWidthCell](#fixedwidthcell) whose width is set based on the size
- * of grid currently being displayed. Used for spacing out cells from each
- * other. The gutter itself doesn't hold any content, it just spaces it out.
+ * Gutter is a private component.
```

Comment:
Should we specify what it is private to? Like "private to the grid package" or "private to the Grid component"?